docs/UserGuide.md

```diff
@@ -127,9 +124,18 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
-
-Format: `add -n NAME -p PHONE_NUMBER -e EMAIL -a ADDRESS [t/TAG]…​`
+```
+add -n NAME -p PHONE_NUMBER -e EMAIL -a ADDRESS [t/TAG]…​
+```
+Adds a new person to the address book.
+
+| Param            | Remarks                                                                                                        |
```

Comment:
The table makes it much more easier to note down the information and flags necessary